geo/src/algorithm/chamberlain_duquette_area.rs

```diff
@@ -49,7 +46,7 @@ use num_traits::Float;
 /// ```
 pub trait ChamberlainDuquetteArea<T>
 where
-    T: Float + CoordNum,
+    T: CoordFloat,
```

Comment:
The CoordFloat additions are an unrelated no-op cleanup. I just prefer to uniformly reference our internal types over the num_types.